docs/aws_sso.md

```diff
@@ -0,0 +1,56 @@
+If you are working with AWS, there are two things you should avoid at all costs:
+
+* Use account’s root credentials for day-to-day work
+* Provision individual users in your account and generate credentials for them
+
+AWS Single Sign On (SSO) with Athenz allows you to avoid both. Because an Athenz
+domain is directly related to an AWS account, the same authorization mechanism
+can be leveraged for SSO to the AWS console, via SAML-based integration. The
+Identity Provider service validates the user and then uses Athenz
+to determine the AWS role(s) the user can assume. This set of roles also indirectly
+identifies all the accounts the user can access from the console. In its simplest
+form, the single (potentially large) list can be presented to AWS in a SAML
+document, and AWS allows selection of the account/role to use.
+
+## Onboard to AWS
+
+To register an AWS Account with a domain, run the following command:
+```
+zms-cli -d <domain-name> set-aws-account <aws-account-id>
+```
+
+In the Athenz UI, when you select your domain,
+you should see your onboarded AWS account number.
```

Comment:
Removed the following text:
" Typically you create a separate AWS account for each one of your environments
 for your product - e.g. athenz.prod, athenz.dev and athenz.stage. Additionally,
 you can also request an AWS account for your personal domain - `home.<okta-id>`.
 This allows developers to test out any specific features without affecting
 their product environments.

 Once you have created your AWS account, it will be automatically onboarded to
 the Athenz domain you specified as part of your AWS Account Create request.

 In the [Athenz UI](https://ui.athenz.ouroath.com/athenz), when you select your domain,
 you should see your onboarded AWS account number. If you need to make any
 changes to your setup (e.g. remove or change AWS account association), one of
 the domain admins must file a ticket in the Jira
 [ATHENS project](https://jira.ouroath.com/secure/CreateIssue.jspa?pid=10388&issuetype=10100)."